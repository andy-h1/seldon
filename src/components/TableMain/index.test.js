import React from 'react';
import { render, waitFor } from '@testing-library/react';
import { TableMain } from '.';

describe('Main Table component', () => {
  it('should render', async () => {
    const { container } = render(<TableMain />);
    await waitFor(() => expect(container.firstChild).toMatchSnapshot());
  });
});
