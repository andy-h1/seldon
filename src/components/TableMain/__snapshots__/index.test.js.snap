// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Table component should render 1`] = `
<div
  class="makeStyles-root-1"
>
  <div
    class="MuiTableContainer-root"
  >
    <table
      aria-label="enhanced-table"
      aria-labelledby="tableTitle"
      class="MuiTable-root"
    >
      <thead
        class="MuiTableHead-root sc-bdfBwQ QcFOd"
      >
        <tr
          class="MuiTableRow-root MuiTableRow-head"
        >
          <th
            class="MuiTableCell-root MuiTableCell-head sc-gsTCUz ALHlQ MuiTableCell-alignLeft MuiTableCell-paddingNone"
            scope="col"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-3"
              role="button"
              tabindex="0"
            >
              Campaign
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            aria-sort="ascending"
            class="MuiTableCell-root MuiTableCell-head sc-gsTCUz ALHlQ MuiTableCell-alignRight"
            scope="col"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-3 MuiTableSortLabel-active"
              role="button"
              tabindex="0"
            >
              Efficiency
              <span
                class="makeStyles-visuallyHidden-2"
              >
                sorted ascending
              </span>
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            class="MuiTableCell-root MuiTableCell-head sc-gsTCUz ALHlQ MuiTableCell-alignRight"
            scope="col"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-3"
              role="button"
              tabindex="0"
            >
              Plays
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            class="MuiTableCell-root MuiTableCell-head sc-gsTCUz ALHlQ MuiTableCell-alignRight"
            scope="col"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-3"
              role="button"
              tabindex="0"
            >
              Finished
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            class="MuiTableCell-root MuiTableCell-head sc-gsTCUz ALHlQ MuiTableCell-alignRight"
            scope="col"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-3"
              role="button"
              tabindex="0"
            >
              Likes
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            class="MuiTableCell-root MuiTableCell-head sc-gsTCUz ALHlQ MuiTableCell-alignRight"
            scope="col"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-3"
              role="button"
              tabindex="0"
            >
              Comments
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
        </tr>
      </thead>
      <tbody
        class="MuiTableBody-root sc-dlfnbm hEGGFo"
      >
        <tr
          class="MuiTableRow-root"
        >
          <th
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-paddingNone"
            role="cell"
            scope="row"
          >
            Social Networks traffic
          </th>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            95
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            95
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            31
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            11
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <th
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-paddingNone"
            role="cell"
            scope="row"
          >
            Instagram stories marketing
          </th>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            186
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            186
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            92
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            8
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            1
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <th
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-paddingNone"
            role="cell"
            scope="row"
          >
            Visitors from video sources
          </th>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            329
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            329
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            256
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            32
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            4
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <th
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-paddingNone"
            role="cell"
            scope="row"
          >
            Other Referrers
          </th>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            804
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            804
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            697
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            40
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body sc-hKgILt blMROb MuiTableCell-alignRight"
          >
            22
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
